README.md

```diff
@@ -106,7 +106,50 @@ use {
 
 ## Usage
 
-### list
+### Global Configuration
```

Comment:
@Chetan496 feel free to point out anything unclear in that section or anything that does not work for you.

Reply:
> @Chetan496 feel free to point out anything unclear in that section or anything that does not work for you. (this is re: #38)

Thanks for this, this is very helpful.
I will try this feature soon.